src/devices/Amg88xx/StatusFlagBit.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the status bits of the status register (addr: 0x04)
+    /// </summary>
+    [Flags]
+    public enum StatusFlagBit : byte
```

Comment:
+1 for visibility

Reply:
Can be internal. I use the InternalsVisibleToAttribute to expose it to the unit test assembly.